src/admin/components/elements/DatePicker/DatePicker.tsx

```diff
@@ -49,6 +50,22 @@ const DateTime: React.FC<Props> = (props) => {
     else dateTimeFormat = 'MMM d, yyy';
   }
 
+  const years = Array.from({ length: getYear(new Date()) - 1990 + 1 }, (_, i) => i + 1990);
```

Comment:
Is this limiting the dropdown to no dates before 1990?
I haven't pulled this down yet.

Reply:
Yep! Doesn't affect what dates are possible - just what is shown in the Dropdown.
Should I lower this?

Reply:
I pulled it down. The dropdown is very effective for getting to a certain year, as long as it is range. The more concerning thing I am seeing is that using the arrows, I can't choose a date outside of the dropdown year range.

For example if I am at Dec 2023 and click the right arrow, the picker now jumps to Jan 1990.
![image](https://user-images.githubusercontent.com/6434612/226972850-c95e03b6-1904-43ed-ada6-047cb2364019.png)
![image](https://user-images.githubusercontent.com/6434612/226972874-2ca7f411-0091-4a9d-843f-b3c03d6aeb6f.png)